include/vapor/FlowParams.h

```diff
@@ -53,35 +53,94 @@ class PARAMS_API FlowParams : public RenderParams {
+    //! vectors are in meters, you can apply a velocity multiplyer of 1000 to correct the mismatch.
+    //! \param[in] double - Velocity field multiplier for flow rendering
     void   SetVelocityMultiplier(double);
 
+    //! Get the number of steps to advect a steady flow line (aka a streamline)
+    //! \retval long - The number of steps a steady flow line is advected
     long GetSteadyNumOfSteps() const;
+    
+    //! Set the number of steps to advect a steady flow line (aka a streamline)
+    //! \param[in] long - The number of steps a steady flow line is advected
     void SetSteadyNumOfSteps(long);
 
+    //! Get the mode for generating seeds (points of origin) for the flow renderer.
+    //! \retval int - The current seed generation mode for the flow renderer. 0 = Gridded, 1 = Random, 2 = Random with bias, 3 = List of seeds
     int  GetSeedGenMode() const;
+    
+    //! Set the mode for generating seeds (points of origin) for the flow renderer.
+    //! \param[in] int - The current seed generation mode for the flow renderer. 0 = Gridded, 1 = Random, 2 = Random with bias, 3 = List of seeds
     void SetSeedGenMode(int);
 
+    //! Enable or disable the writing of flow renderer data values o a text file.
+    //! \param[in] bool - Enable (1/true) or disable (0/false) the writing of trajectory data values to a text file.
     void SetNeedFlowlineOutput(bool);
+    
+    //! Inquire whether the writing of flow renderer data values are being written to a text file.
+    //! \retval bool - Enable (1/true) or disable (0/false) the writing of trajectory data values to a text file.
     bool GetNeedFlowlineOutput() const;
 
+    //! Get the current flow renderer's advection direction.
+    //! \retval int - The advection direction for the current flow renderer.  (0 = forward, 1 = backward, 2 = bi-directional)
     int  GetFlowDirection() const;
+    
+    //! Set the current flow renderer's advection direction.
+    //! \param[in] int - The advection direction for the current flow renderer.  (0 = forward, 1 = backward, 2 = bi-directional)
     void SetFlowDirection(int);
 
+    //! Get the file name/path to a file containing a list of seed points to advect from.  See https://vapor.readthedocs.io/en/readthedocs/usage/flowRenderer.html#seed-distribution-settings
+    //! \retval string - A file path containing a defined list of seed points to advect from
     std::string GetSeedInputFilename() const;
-    void        SetSeedInputFilename(const std::string &);
+    
+    //! Set the file name/path to a file containing a list of seed points to advect from.  See https://vapor.readthedocs.io/en/readthedocs/usage/flowRenderer.html#seed-distribution-settings
+    //! \param[in] string - A file path containing a defined list of seed points to advect from
+    void SetSeedInputFilename(const std::string &);
 
+    //! If GetNeedFlowlineOutput() returns true, this will return the file path to the text file that data will be written to.
```

Comment:
A small nit: won't this function return the output file name regardless of the value of GetNeedFlowlineOutput()?

Reply:
Yes I suppose it does.  Fixed.

Reply:
It is correct. I've clarified it.